tests/pipeline/test_factor.py

```diff
@@ -585,6 +586,34 @@ def test_returns(self, seed_value, window_length):
 
         check_allclose(expected, out)
 
+    @parameterized.expand([
+        (100, 15),
+        (101, 4),
+        (102, 100),
+        ])
+    def test_percentchange(self, seed_value, window_length):
+
+        pct_change = PercentChange(
+            inputs=(), 
+            window_length=window_length
```

Comment:
Another small style note, I like to leave trailing commas at the end of multiline calls of functions, import lines, or lists so that if a new parameter ever got added the git diff would only be +1 line instead of -1/+2 lines. I also tend to think it looks nicer 🙂